guides/common/assembly_using-kernelcare.adoc

```diff
@@ -0,0 +1,46 @@
+include::modules/con_using-foreman-kernelcare-plug-in.adoc[]
+
+include::modules/proc_installing-kernelcare-plug-in.adoc[leveloffset=+1]
+
+ifdef::katello[]
+:os_dist: stable
+:os_major: 11
+:os_name: Debian
+include::modules/proc_creating-kernelcare-repositories-deb.adoc[leveloffset=+1]
+
+:os_dist: stable
```

Comment:
Debian 10 is now `old-stable`

Reply:
I'm guessing it's normally used as `deb https://repo.cloudlinux.com/kernelcare-debian/10/ stable main`. Would it then make sense to rename it to `kernelcare_os_dist`?

Reply:
Yes. Renamed the attribute.